app/scripts/app.js

```diff
@@ -48,15 +48,15 @@ angular.module('yourStlCourts').config(function($stateProvider, $urlRouterProvid
       }
     })
     .state('courtSearchInfo', {
-      url: '/courtSearch/info',
+      url: '/courtSearch/info?id',
```

Comment:
It's really a court info page, not a "courtSearch" page. "/courts/:id" or "/courts/:id/info" is more reflective of that.